tests/framework/integration/cluster.go

```diff
@@ -260,13 +260,12 @@ func (c *Cluster) ProtoMembers() []*pb.Member {
 }
 
 func (c *Cluster) mustNewMember(t testutil.TB) *Member {
-	memberNumber := c.LastMemberNum
+	uniqueNumber := atomic.AddInt32(&UniqueNumber, 1)*10 + int32(c.LastMemberNum)
```

Comment:
Is this change needed?

Reply:
Yes, otherwise the names of unix sockets will collide in some cases